src/web/toga_web/app.py

```diff
@@ -13,33 +14,246 @@ def __init__(self, interface):
         self.interface = interface
         self.interface._impl = self
 
-        self.create()
-
     def create(self):
-        self.interface.startup()
 
         self.interface.icon.bind(self.interface.factory)
         # self.resource_path = os.path.dirname(os.path.dirname(NSBundle.mainBundle.bundlePath))
 
         formal_name = self.interface.formal_name
+
         self.interface.commands.add(
+            # ---- Help menu ----------------------------------
             toga.Command(
-                self.interface.about_command,
-                'About ' + formal_name,
-                group=toga.Group.APP
+                lambda _: self.interface.about(),
+                "About " + formal_name,
+                group=toga.Group.HELP,
+            ),
+            toga.Command(
+                None,
+                "Preferences",
+                group=toga.Group.HELP,
```

Comment:
Doesn't look as if this is being used anywhere.

Reply:
Correct - it's a placeholder that acts as a tease for future functionality (in the hope it will jolt someone into potentially contributing).